adds documentation for sharepoint lists (#5099)
alcionai/corso: website/docs/support/known-issues.md

  updated the locale since the backup was created.

* In-place Exchange contacts restore will merge items in folders named "Contacts" or "contacts" into the default folder.

> * Sharepoint list item "attachments" aren't Backup-ed up, Restore-d and Export-ed.

Reviewer: no hyphens here, please.
```suggestion
* Sharepoint list item "attachments" aren't backed up, restored or exported.
```

  Graph API doesn't currently provide attachment data for Lists or List Items.

>   An [issue](https://github.com/alcionai/corso/issues/5085) has been created to track the progress..

Reviewer 2: We don't need to mention particular issues here, as those are for internal tracking.

```suggestion
```

>   This is because of limitation of GRAPH APIs to provide the relevant info in responses.
>   An issue has been created to track the progress: https://github.com/alcionai/corso/issues/5085

Reviewer: ```suggestion
  Graph API does not currently provide attachment data for Lists or List Items.
  An issue has been created to track the progress: https://github.com/alcionai/corso/issues/5085
```
